css/ampshared.css

```diff
@@ -449,7 +449,7 @@ amp-accordion > section {
 }
 
 /* Collapse content by default. */
-amp-accordion > section > :last-child {
+amp-accordion > section > :last-child:not([rendersubtree]) {
```

Comment:
Back to this item: can we remove the `:not([rendersubtree])` somehow here? Let's instead move `display: block` to the `amp-accordion > section:not([expanded]) > :last-child[rendersubtree]` selector in the amp-accordion.css

Reply:
Yes, `:not([rendersubtree])` shouldn't be needed once we restore `display: block` to the collapsed/activateable section content.